tests/rules/exporter/test_rules2dms.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from cognite.neat.rules.exporter.dm.rules2dms import DataModel
+from cognite.neat.rules._exceptions import Error10
+
+
+def test_rules2dms(simple_rules):
+    data_model = DataModel.from_rules(transformation_rules=simple_rules)
+
+    assert len(data_model.containers) == 4
+    assert len(data_model.views) == 4
+    assert list(data_model.views.keys()) == ["CountryGroup", "Country", "PriceArea", "PriceAreaConnection"]
+    assert list(data_model.containers.keys()) == ["CountryGroup", "Country", "PriceArea", "PriceAreaConnection"]
```

Comment:
```suggestion
    assert list(data_model.views) == ["CountryGroup", "Country", "PriceArea", "PriceAreaConnection"]
    assert list(data_model.containers) == ["CountryGroup", "Country", "PriceArea", "PriceAreaConnection"]
```

Keys are default for dicts